web/js/map/wv.map.layerbuilder.js

```diff
@@ -149,12 +149,19 @@ wv.map.layerbuilder = wv.map.layerbuilder || function(models, config, cache, Par
         if ( !matrixSet ) {
             throw new Error(def.id + ": Undefined matrix set: " + def.matrixSet);
         }
-        matrixIds = [];
-        _.each(matrixSet.resolutions, function(resolution, index) {
-            matrixIds.push(index);
-        });
+		if ("undefined" === typeof def.matrixIds) {
+			matrixIds = [];
+			_.each(matrixSet.resolutions, function(resolution, index) {
+				matrixIds.push(index);
+			});
+		}
+		else {
+			matrixIds = def.matrixIds;
+		}
+
         extra = "";
-        if(day) {
+
+		if(day) {
```

Comment:
@rboller Just some extra indentation at lines 164 and 152-160. We are going to automate the indentation soon so it isn't a big deal. All else looks good!